libqtile/widget/clock.py

```diff
@@ -58,6 +46,8 @@ class Clock(base.InLoopPollText):
     def __init__(self, **config):
         base.InLoopPollText.__init__(self, **config)
         self.add_defaults(Clock.defaults)
+        if pytz and isinstance(self.timezone, str):
```

Comment:
Then here, `pytz is not None`

Reply:
We may want to log something, one way or another, I'm not familiar with pytz enough to know if it could lead to some unexpected behavior if someone didn't realize that pytz was doing something to the timezone, or if someone expects pytz to resolve the timezone but it silently doesn't because it is not installed.

Reply:
Done!